apps/functions-subscription/api/internal.yaml

```diff
@@ -391,15 +415,57 @@ components:
           $ref: '#/components/schemas/TrialName'
         description:
           $ref: '#/components/schemas/TrialDescription'
+    TrialState:
+      type: string
+      description: |-
+        - CREATING: The Trial System is creating the resources related to the trial.
+          The configuration is not yet completed.
+        - CREATED: The Trial System created the resources for the trial.
+      enum:
+        - CREATING
+        - CREATED
+    CreatedTrial:
+      type: object
+      required:
+        - id
+        - state
+      properties:
+        id:
+          $ref: '#/components/schemas/TrialId'
+        state:
+          $ref: '#/components/schemas/TrialState'
     Trial:
       type: object
       required:
         - id
         - name
+        - state
       properties:
         id:
           $ref: '#/components/schemas/TrialId'
         name:
           $ref: '#/components/schemas/TrialName'
         description:
           $ref: '#/components/schemas/TrialDescription'
+        state:
+          $ref: '#/components/schemas/TrialState'
+        queue:
```

Comment:
**suggestion**: Rename it to something more generic, and since they are specific to Azure, I would move them into an Azure object or something similar. E.g.:

```
channel: {
  azure: {
    queueName: '',
    manageIdentityId: ''
  }
}
```

Reply:
I'm going to to this in a separate PR (https://github.com/pagopa/trial-system/pull/107#discussion_r1658396614)